CRM/Event/BAO/Event.php

```diff
@@ -2342,18 +2342,10 @@ public static function getICalLinks($eventId = NULL) {
   }
 
   /**
-   * @param array $profileIds
```

Comment:
@MegaphoneJon did you mean to remove the comment block?

Reply:
Yes - the type hints make it redundant.  If you feel strongly, I can put them back and update the return value.

Reply:
hmm - we should really get coder updated & define our position on this - in the mean time I'm OK with playing push-me-pull-me depending on the IDE settings a dev is using